test/helpers/helpers.js

```diff
@@ -1,5 +1,42 @@
-var marked = require('../../lib/marked.js');
+const marked = require('../../');
+const HtmlDiffer = require('@markedjs/html-differ').HtmlDiffer;
```

Comment:
```suggestion
const { HtmlDiffer } = require('@markedjs/html-differ');
```

Might as well embrace ES6 😄

Reply:
That doesn't work in node 4